Makefile

```diff
@@ -17,17 +17,13 @@ find-ignored-tests: ## find all ignored tests (e.g. fdescribe)
 # Documentation targets
 #################################
 
-.PHONY: docs-publish
-docs-publish: ## build and publish docs to github Pages
-	mkdocs gh-deploy
-
 .PHONY: docs-build
 docs-build: ## build docs into the local 'site' folder
-	mkdocs build
+	mkdocs build --strict
```

Comment:
The `--strict` flag is necessary, so that warnings are not silently ignored. Often, those warnings later create problems when I want to publish the docs to docs.dasch.swiss. So it is better to catch them early, and let the CI fail if problematic markdown files are checked in.